### ISSUE TITLE
Jaws is interpreting checkboxes as ReadOnly

### ISSUE THREAD

**Issue by tyaychuk** [vakata/jstree#856]
Hi Ivan,

I am impressed with your jstree. Quite the masterpiece.

Unfortunately I am having issues getting jaws screen reader (V13) to interpret the tree if it has checkboxes. I tried jaws on your website (2nd box on the right hand side of http://www.jstree.com/). It doesn't read the text of the item, but just says "readonly"  Do you know if there is an ARIA directive that can help with this?

Thanks,
Tim.
